tariff-logic/taric-footwear.test.js

```diff
@@ -15,674 +15,1000 @@ test.each([
+  ],
+  [
+    '640359',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'yes' },
+      { questionKey: 'heightOfSoleAndHeel', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'no' }
+    ],
+    '6403593100'
+  ],
+  [
+    '640359',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'yes' },
+      { questionKey: 'heightOfSoleAndHeel', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' }
+    ],
+    { code: '640359', questionKey: 'genderType' }
+  ],
+  [
+    '640359',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'yes' },
+      { questionKey: 'heightOfSoleAndHeel', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' },
+      { questionKey: 'genderType', answerKey: 'men' }
+    ],
+    '6403593500'
+  ],
+  [
+    '640359',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'yes' },
+      { questionKey: 'heightOfSoleAndHeel', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' },
+      { questionKey: 'genderType', answerKey: 'women' }
+    ],
+    '6403593900'
+  ],
+  // Vamp = no
+  [
+    '640359',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' }
+    ],
+    { code: '640359', questionKey: 'slippers' }
+  ],
+  [
+    '640359',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'yes' }
+    ],
+    '6403595000'
+  ],
+  [
+    '640359',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' }
+    ],
+    { code: '640359', questionKey: 'lengthOfInsole' }
+  ],
+  [
+    '640359',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'no' }
+    ],
+    '6403599100'
+  ],
+  [
+    '640359',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' }
+    ],
+    { code: '640359', questionKey: 'genderType' }
+  ],
+  [
+    '640359',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
+      { questionKey: 'vamp', answerKey: 'no' },
+      { questionKey: 'slippers', answerKey: 'no' },
+      { questionKey: 'lengthOfInsole', answerKey: 'yes' },
+      { questionKey: 'genderType', answerKey: 'men' }
+    ],
+    '6403599500'
+  ],
+  [
+    '640359',
+    [
+      { questionKey: 'madeOnBase', answerKey: 'no' },
```

Comment:
Same here, lack test case with genderType => other that also ends up as 6403599900